include/n64dd.h

```diff
@@ -1,10 +1,15 @@
 #ifndef N64DD_H
 #define N64DD_H
 
+#include "z64play.h"
```

Comment:
I would suggest forward declaring `PlayState` instead of pulling the whole header since only a pointer to the struct is needed.

Reply:
Then it wouldn't pull in the typedef either which I don't really like, but sure I guess it's already like that in other places